# HG changeset patch
# User Rik <rik@octave.org>
# Date 1473256518 25200
#      Wed Sep 07 06:55:18 2016 -0700
# Node ID d77caa2d523d2c1a428678434ad19b6bec2788f0
# Parent  70aa2b362a311ef4ec1aa3cfe1a11ddd86758a98
CITATION: update for 4.2.0 and use @manual in Bibtex (bug #47058).

* CITATION: update for 4.2.0 and use @manual in Bibtex (bug #47058).

diff --git a/CITATION b/CITATION
--- a/CITATION
+++ b/CITATION
@@ -1,18 +1,18 @@
 To cite GNU Octave in publications use:
 
-  John W. Eaton, David Bateman, Søren Hauberg, Rik Wehbring (2015).
-  GNU Octave version 4.0.0 manual: a high-level interactive language for
+  John W. Eaton, David Bateman, Søren Hauberg, Rik Wehbring (2016).
+  GNU Octave version 4.2.0 manual: a high-level interactive language for
   numerical computations.
   URL http://www.gnu.org/software/octave/doc/interpreter/
 
 A BibTeX entry for LaTeX users is:
 
-  @book{,
+  @manual{,
+    title     = {{GNU Octave} version 4.2.0 manual: a high-level interactive language for numerical computations},
     author    = {John W. Eaton, David Bateman, S\oren Hauberg, and Rik Wehbring},
-    title     = {{GNU Octave} version 4.0.0 manual: a high-level interactive language for numerical computations},
-    year      = {2015},
+    year      = {2016},
     url       = {http://www.gnu.org/software/octave/doc/interpreter},
   }
 
 We have invested a lot of time and effort in creating GNU Octave, please cite it
 when using it.  See also `citation pkgname' for citing Octave packages.
